README.md

```diff
@@ -90,6 +90,78 @@ windows_batch 'echo some env vars' do
 end
 ```
 
+### windows_certificate
+
+Installs a certificate into the Windows certificate store from a file, and grants read-only access to the private key for designated accounts
+
+#### Actions
+- :create: creates or updates a certificate
+- :delete: deletes a certificate
+- :acl_add: adds read-only entries to a certificte's private key ACL
+
+#### Attribute Parameters
+- source: name attribute. The source file (for create and acl_add) or thumprint/subject (for delete and acl_add).
+- pfx_password: the password to access the source if it is a pfx file.
+- private_key_acl: array of 'domain\account' entries to be granted read-only access to the certificte's private key. This is not idempotent.
+- store_name: the certificate store to maniplate. One of MY (default : personal store), CA (trusted intermediate store) or ROOT (trusted root store)
+- user_store: if false (default) then use the local machine store; if true then use the current user's store.
+
+#### Examples
+```ruby
+# Add PFX cert to local machine personal store and grant accounts read-only access to private key
+windows_certificate "c:/test/mycert.pfx" do
+	pfx_password	"password"
+	private_key_acl	["acme\fred", "pc\jane"]
+end
+```
+
+```ruby
+# Add cert to trusted intermediate store
+windows_certificate "c:/test/mycert.cer" do
+	store_name	"CA"
+end
+```
+
+```ruby
+# Remove all certicates matching the subject
+windows_certificate "me.acme.com" do
+	action :delete
+end
+```
+
+### windows_certificate_binding
+
+Binds a certificate to an HTTP port in order to enable TLS communication.
```

Comment:
Hmm - wouldn't this be more in place within the IIS or windows_firewall cookbooks?